content/en/events/2022/kcseu/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Contributor Summit Europe 2022"
+type: docs
+aliases: [ "/summit", "/kcseu2022" ]
+weight: 1
+description: >
+  Kubernetes Contributor Summit Europe, hosted in Valencia Spain.
+
+---
+
+
+### About
+
+The Europe Summit will be a lightweight event taking place on May 16th
+in Valencia, Spain alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>.
+
+The Summit will be a loosely organized event with a [relaxed schedule](/events/kcseu/schedule),
```

Comment:
All file paths need to be updated
Find -> Replace
`/events/kcseu` -> `/events/2022/kcseu`